bin/graderutils

```diff
@@ -1,4 +1,65 @@
 #!/bin/sh
 export PYTHONPATH=/exercise
-[ "$*" ] || set -- /exercise/test_config.yaml
-exec capture -e /feedback/grading-script-errors -- _graderutils_exec "$@"
+export EXERCISE_PATH=/exercise
+
+use_iotester=false
+use_rpyc=false
+while [ "$1" ]; do
```

Comment:
For the future, it might be more robust to test argument list size `while [ $# -gt 0 ]`, see https://github.com/apluslms/grading-base/blob/master/bin/asuser#L4